sunpy/coordinates/ephemeris.py

```diff
@@ -215,12 +215,16 @@ def _sun_north_angle_to_z(frame):
     and observation time.
     """
     # Find the Sun center in HGS at the frame's observation time
-    sun_center = SkyCoord(0*u.deg, 0*u.deg, 0*u.km, frame=HGS, obstime=frame.obstime)
+    sun_center_repr = SphericalRepresentation(0*u.deg, 0*u.deg, 0*u.km)
+    sun_center = SkyCoord(sun_center_repr._apply('repeat', frame.obstime.size),
```

Comment:
a couple of descriptive comments about why the repeat here would be useful. It's a little over my head and we were talking about it on matrix the other day.